slave/url_creator.py

```diff
@@ -8,9 +8,11 @@
 
 
 class UrlCreator(object):
-    def __init__(self, config, repo):
+    def __init__(self, config, repo, other_platform):
```

Comment:
For me other_platform should be an optional parameter (e.g. other_platform=None).